idea_town/frontend/static-src/app/views/feedback-view.js

```diff
@@ -56,9 +62,9 @@ export default BaseView.extend({
     // Hide the feedback form right away & let the submission happen in the
     // background - user probably doesn't care about progress or confirmation
     this.animateRemove();
+    this.parentUninstall(this.parentUpdateAddon, this.parentModel);
```

Comment:
This survey view is also feedback view is also used for the surveys while the experiment is installed & in progress. It should *not* be uninstalling addons.